### PR TITLE
Simplify task fetching to avoid timeout in strapi v3

### PR DESCRIPTION
**Pull request by bidoubiwa**
Tests in cypress were failing because of a weird tasks listener. 

I simplified it by removing the complex system to listen to tasks and replacing it with a setInterval running until no tasks are left enqueued or processing.

### CODE REVIEW

File: admin/src/components/Collections.js [hunk 1]
```diff
@@ -200,12 +112,10 @@ const Collections = ({ updateCredentials }) => {
...
-      // Start watching collections that have pending tasks
-      collections.map(col => {
-        if (col.isIndexing) {
-          watchTasks({ collection: col.collection })
-        }
-      })
+      const isIndexing = collections.find(col => col.isIndexing === true)
+
+      if (!isIndexing) setRealTimeReports(false)
+      else setRealTimeReports(true)
```
Comment (mdubus) [inline admin/src/components/Collections.js:118]: Can't you just do `setRealTimeReports(isIndexing)` ?

File: connectors/meilisearch/index.js [hunk 2]
```diff
@@ -76,6 +76,8 @@ module.exports = async ({ storeConnector, collectionConnector }) => {
      */
     waitForTask: async function ({ collection, taskUid }) {
       try {
+        console.log({ collection, taskUid })
```
Comment (mdubus) [inline connectors/meilisearch/index.js:79]: 👀

File: admin/src/components/Collections.js (hunk 1 above)
Comment (bidoubiwa) [reply admin/src/components/Collections.js:118]: Absolutely 😂 I was tired

File: connectors/meilisearch/index.js (hunk 2 above)
Comment (bidoubiwa) [reply connectors/meilisearch/index.js:79]: The whole function should not even be there! Thanks

**Review by mdubus — APPROVED** [review]
LGTM ✨ 🦕

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**